SimTracker/TrackerHitAssociation/plugins/trackerHitAssociationHeterogeneousProduct.h

```diff
@@ -21,8 +21,10 @@ namespace trackerHitAssociationHeterogeneousProduct {
 
   struct ClusterSLGPU {
 
+   using Clus2TP = std::array<uint32_t,6>;
```

Comment:
Please use a more meaningful or at least less abbreviated name, for example 'ClusterToTP' (assuming that is the meaning).

Reply:
What is the meaning of `6` ?